### PR TITLE
Add links to linked objects on the measure details tab

### PR DESCRIPTION
**Pull request by LaurenQurashi**
# TP-2000-971 Add links to linked objects on the measures detail tab page

## Why
A measure is linked with a bunch of other things viewable objects on TAP, so it makes sense to have links to see them easily. 

## What
This PR adds links to the measures details (core data) tab for: 
- Measure generating regulation,  and adds the title too.. 
- Commodity Code
- Additional Code
- Geographical area
- Geographical exclusions
- Quota order number 

## Checklist
- Requires migrations? - no
- Requires dependency updates? - no

Before: 
Screenshot 2023-07-28 at 15 49 40

After: 
Screenshot 2023-07-28 at 15 47 48

### CODE REVIEW

File: measures/jinja2/includes/measures/tabs/core_data.jinja [hunk 1]
```diff
@@ -44,22 +44,22 @@
...
         {
             "key": {"text": "Geographical area"},
-            "value": {"text":  object.geographical_area.area_id ~ " - " ~ object.geographical_area.get_description().description if object.geographical_area else '-'},
+            "value": {"text": create_link(url("geo_area-ui-detail", kwargs={"sid": object.geographical_area.sid}), object.geographical_area.get_description().description) if object.geographical_area else "-"},
             "actions": {"items": []}
         },
         {
             "key": {"text": "Geographical area exclusions"},
-            "value": {"text":  geographical_exclusions },
+            "value": {"text": geographical_exclusions if object.exclusions.current() else "-"},
```
Comment (dalecannon) [inline measures/jinja2/includes/measures/tabs/core_data.jinja:57]: This conditional is duplicated in the `geographical_exclusions` set assignment.  I wonder if either one could be removed to save a query.

File: measures/jinja2/includes/measures/tabs/core_data.jinja [hunk 2]
```diff
@@ -44,22 +44,22 @@
...
         {
             "key": {"text": "Additional code"},
-            "value": {"text":  object.additional_code.type.sid ~ object.additional_code.code if object.additional_code else '-'},
+            "value": {"text": create_link(url("additional_code-ui-detail", kwargs={"sid": object.additional_code.sid}), object.additional_code.sid) if object.additional_code else "-"},
             "actions": {"items": []}
         },
         {
             "key": {"text": "Geographical area"},
-            "value": {"text":  object.geographical_area.area_id ~ " - " ~ object.geographical_area.get_description().description if object.geographical_area else '-'},
+            "value": {"text": create_link(url("geo_area-ui-detail", kwargs={"sid": object.geographical_area.sid}), object.geographical_area.get_description().description) if object.geographical_area else "-"},
```
Comment (dalecannon) [inline measures/jinja2/includes/measures/tabs/core_data.jinja:52]: Previously the description was prefixed with the geographical area `area_id`. Would it still be useful to include that?

**Review by dalecannon — APPROVED** [review]

File: measures/jinja2/includes/measures/tabs/core_data.jinja (hunk 2 above)
Comment (LaurenQurashi) [reply measures/jinja2/includes/measures/tabs/core_data.jinja:52]: Good spot! Originally it was an accident but Steve said he thinks it's neater to leave it out 🎉

File: measures/jinja2/includes/measures/tabs/core_data.jinja (hunk 1 above)
Comment (LaurenQurashi) [reply measures/jinja2/includes/measures/tabs/core_data.jinja:57]: Thank you! Great spot 🎉

**Pull request merged**